### PR TITLE
[22] Newsletter Form Validation

### PR DESCRIPTION
**Pull request by MYKEL7**
## Task: `Newsletter Form Validation`
- `Validated the newsletter form for the three pages`

### CODE REVIEW

File: submissions/onyemeMichael/css/contact.css [hunk 1]
```diff
@@ -96,6 +96,19 @@ body {
     overflow: hidden;
  }
 
+ .error-message {
+   color:yellow;
```
Comment (TundeOladejo) [inline submissions/onyemeMichael/css/contact.css:100]: I will suggest you use color `red` to represent your error messages.

File: submissions/onyemeMichael/css/signUp.css [hunk 2]
```diff
@@ -98,6 +98,13 @@ footer nav {
    overflow: hidden;
 }
 
+.error-message {
+   color:yellow;
```
Comment (TundeOladejo) [inline submissions/onyemeMichael/css/signUp.css:102]: Here too. Use `red`

File: submissions/onyemeMichael/css/product.css [hunk 3]
```diff
@@ -141,6 +141,13 @@ a{
     color: white;
     text-decoration: none;
  }
+
+ .error-message {
+    color: yellow;
```
Comment (TundeOladejo) [inline submissions/onyemeMichael/css/product.css:146]: Change the color here too.

**Comment by MYKEL7** [comment]
Done sir

**Review by adaugochi — APPROVED** [review]

**Pull request merged**